TTS/tts/utils/text/__init__.py

```diff
@@ -51,36 +52,44 @@ def text2phone(text, language, use_espeak_phonemes=False):
         ph = japanese_text_to_phonemes(text)
         return ph
 
-    if gruut.is_language_supported(language):
-        # Use gruut for phonemization
-        phonemizer_args = {
-            "remove_stress": True,
-            "ipa_minor_breaks": False,  # don't replace commas/semi-colons with IPA |
-            "ipa_major_breaks": False,  # don't replace periods with IPA ‖
-        }
-
-        if use_espeak_phonemes:
-            # Use a lexicon/g2p model train on eSpeak IPA instead of gruut IPA.
-            # This is intended for backwards compatibility with TTS<=v0.0.13
-            # pre-trained models.
-            phonemizer_args["model_prefix"] = "espeak"
-
-        ph_list = gruut.text_to_phonemes(
-            text,
-            lang=language,
-            return_format="word_phonemes",
-            phonemizer_args=phonemizer_args,
-        )
-
-        # Join and re-split to break apart dipthongs, suprasegmentals, etc.
-        ph_words = ["|".join(word_phonemes) for word_phonemes in ph_list]
-        ph = "| ".join(ph_words)
-
-        # Fix a few phonemes
-        ph = ph.translate(GRUUT_TRANS_TABLE)
-        return ph
-
-    raise ValueError(f" [!] Language {language} is not supported for phonemization.")
+    if not gruut.is_language_supported(language):
+        raise ValueError(f" [!] Language {language} is not supported for phonemization.")
+
+    # Use gruut for phonemization
+    ph_list = []
+    for sentence in gruut.sentences(text, lang=language, espeak=use_espeak_phonemes):
```

Comment:
TTS used to have similar `phonemization` for times builtin, have those been removed?

Reply:
I am talking about TTS/tts/utils/text/time.py